official/docs/go/referral/create-referral-user.go

```diff
@@ -0,0 +1,27 @@
+package example
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/EasyPost/easypost-go/v2"
+)
+
+func main() {
+	apiKey := os.Getenv("<YOUR_TEST/PRODUCTION_API_KEY>")
+	client := easypost.New(apiKey)
+
+	name := "Test Referral"
+	email := "test@example.com"
+	phone := "5555555555"
+
+	referralUser, _ := client.CreateReferralCustomer(
+		&easypost.UserOptions{
+			Name:  &name,
+			Email: &email,
+			Phone: &phone,
+		},
+	)
```

Comment:
No need for extra lines, just place the values inline.

```suggestion
	referralUser, _ := client.CreateReferralCustomer(
		&easypost.UserOptions{
			Name:  "Test Referral",
			Email: "test@example.com",
			Phone: "5555555555",
		},
	)
```

Reply:
The values in the `UserOptions` are pointers, we can't pass in values directly, we need to dereference them. 

Link to the code: https://github.com/EasyPost/easypost-go/blob/master/user.go#L25-L37